data/blog/2020/2020-03-09-transferring-zf-to-laminas.md

```diff
@@ -32,6 +32,8 @@ use [`replace` in `composer.json`](https://getcomposer.org/doc/04-schema.md#repl
 
 We needed a tool capable of much more than just "rewriting namespaces".
 
+<!--- EXTENDED -->
```

Comment:
This is so folks don't have to scroll forever to see the rest of the posts. :smile: